README.md

```diff
@@ -67,6 +67,20 @@ performance will not be good. It might be a good idea to move this data to e.g.
 1. Run the CI suite
    `bundle exec rake ci`
 
+### Important details for PostgreSQL security and configuration
```

Comment:
Should this be a lower level header? It seems like it's bumping the other subsections of `Installation` down into this section.